app/src/main/res/values/debating_timer.xml

```diff
@@ -141,9 +141,12 @@
     <string name="changelogDialog_message">
           • Debate style formats are now available for download, with only four included in the app.
             You may need to download the formats you were previously using to continue.
+        \n• Added support for multilingual debate formats, with thanks to Frank Richter. (BP is in
```

Comment:
@res2k let me know if you'd like to be credited in some other way!

Reply:
It's fine as it is, thanks.